src/Plugins/SimplnxCore/docs/ExtractPipelineToFileFilter.md

```diff
@@ -0,0 +1,15 @@
+# Extract DREAM.3D Pipeline To File
+
+## Description
+
+This **Filter** reads the pipeline from an hdf5 file with the .dream3d extension and writes it back out to a json formatted pipeline file with the appropriate extension based on whether the pipeline is a legacy pipeline or not. (.json extension for legacy and .d3dpipeline extension otherwise) 
```

Comment:
This **Filter** reads the pipeline from an hdf5 file with the .dream3d extension and writes it back out to a json formatted pipeline file with the appropriate extension based on whether the pipeline is a DREAM.3D version 6 (.json) or DREAM3D-NX version 7 (.d3dpipeline) formatted pipeline.